2017/gonumtour.slide

```diff
@@ -0,0 +1,306 @@
+An Introduction to Gonum
+15 July 2017
+
+Brendan Tracey
+
+* Talk Outline
+- Overview of Gonum 
+- Gonum Philosophy
+- Brief examples of packages
+- Call for help
+
+* Overview of Gonum
+
+* What
+- Set of libraries for scientific and numeric programming (think: Numpy / Scipy)
+- Our packages (July 2017):
+
+.image gonumtour/packages.png 450 _
+
+* Where
+[[https://gonum.org][gonum.org]]
+- New [[https://gohugo.io/][Hugo]] based site coming soon!
+
+	go get -u -t gonum.org/v1/gonum
+
+* Who
+To date: 35 Contributors
+
+We live in New Mexico USA, Adelaide Australia, Karlsruhe Germany, Clermont-Ferrand France
+
+That is, we have self-assembled. Come join us!
+
+- [[https://github.com/gonum][github.com/gonum]]
+- [[https://github.com/gonum/gonum/blob/master/LICENSE][Contributing guide]]
+- [[https://groups.google.com/forum/#!forum/gonum-dev][Mailing list]]
+
+* Goals of Gonum
+
+* Challenge: The Two Language Problem
+- Code that runs fast (C, C++, Fortran)
+- Code that is easy to write (Matlab, R, Python)
+
+	%% Direct Method
+	%     
+	%     for jj = 1:numCities
+	%         B = zeros(numCities,numAircraft); % optimal cost used to fly to that city using airplane kk
+	%         for kk = 1:numAircraft
+	%             B(:,kk) = A(ii-1,:)+(edgeCosts(:,jj,kk))';
+	%         end
+	%         minNewPath = min(min(B));
+	%         A(ii,jj) = min(A(ii-1,jj),minNewPath);
+	%     end
+
+
+	Atmp = A(ii-1,:);
+	A(ii,:) = min([A(ii-1,:);(min(min(Atmp(ones(numCities,1),:,ones(numAircraft,1)) +
+		edgeCostSwap,[],2),[],3))']);
+
+- This one-liner sped up code 10x, but illegible
+
+
+* Other issues
+- Parallel computing
+- Scaling to an ecosystem of packages
+- Imports from a recent project of mine
```

Comment:
Yes. AFAICT one cannot do sub bullets in the present tool. Rephrased to make more clear